src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -425,7 +431,8 @@ export class DictionaryModal extends React.Component {
             <Button
               className="btn btn-outline-info"
               id="addDictionary"
-              onClick={this.onSubmit}
+              onClick={this.addDictionary}
+              disabled={this.state.disableButton}
```

Comment:
@ozezzy, I think it is best to destructure all states;

Reply:
done